src/node_jsvmapi.cc

```diff
@@ -481,13 +481,11 @@ namespace v8impl {
   RETURN_STATUS_IF_FALSE(!((maybe).IsNothing()), (status))
 
 #define NAPI_PREAMBLE(e)                                                \
-  do {                                                                  \
-    CHECK_ARG(e);                                                       \
-    RETURN_STATUS_IF_FALSE(v8impl::TryCatch::lastException().IsEmpty(), \
-      napi_pending_exception);                                          \
-    napi_clear_last_error();                                            \
-    v8impl::TryCatch tryCatch(v8impl::V8IsolateFromJsEnv((e)));         \
-  } while(0)
+  CHECK_ARG(e);                                                         \
+  RETURN_STATUS_IF_FALSE(v8impl::TryCatch::lastException().IsEmpty(),   \
+    napi_pending_exception);                                            \
+  napi_clear_last_error();                                              \
+  v8impl::TryCatch tryCatch(v8impl::V8IsolateFromJsEnv((e)))
```

Comment:
I merged https://github.com/nodejs/abi-stable-node/pull/72, so you'll need to rebase on top of that.

Then you can run `vcbuild release nosign test-addons-abi` to run all the ABI tests, which should be all passing now in this branch.